UI/web-src/ui/components/mainMenu/mainMenuSvgIcon.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2023 Radix IoT LLC. All rights reserved.
+ */
+
+import mainMenuSvgIconTemplate from "./mainMenuSvgIcon.html";
+import automationSvg from "./svgs/svg_automation.svg";
+import administrationSvg from "./svgs/svg_administration.svg";
+import configurationSvg from "./svgs/svg_configuration.svg";
+import dataCenterSvg from "./svgs/svg_data_center.svg";
+import dataCollectionSvg from "./svgs/svg_data_collection.svg";
+import dataPointDetailsSvg from "./svgs/svg_data_point_details.svg";
+import dataDistributionSvg from "./svgs/svg_data-distribution.svg";
+import eventsSvg from "./svgs/svg_events.svg";
+import helpSvg from "./svgs/svg_help.svg";
+import presentationSvg from "./svgs/svg_presentation.svg";
+import systemSvg from "./svgs/svg_system.svg";
+import uiSettingsSvg from "./svgs/svg_UI_settings.svg";
+import watchlistSvg from "./svgs/svg_Watchlist.svg";
+
+class mainMenuSvgIconController {
+  static get $$ngIsClass() {
+    return true;
+  }
+  static get $inject() {
+    return [];
+  }
+
+  constructor() {
+    this.automationSvg = automationSvg;
+    this.administrationSvg = administrationSvg;
+    this.configurationSvg = configurationSvg;
+    this.dataCenterSvg = dataCenterSvg;
+    this.dataCollectionSvg = dataCollectionSvg;
+    this.dataPointDetailsSvg = dataPointDetailsSvg;
+    this.dataDistributionSvg = dataDistributionSvg;
+    this.eventsSvg = eventsSvg;
+    this.helpSvg = helpSvg;
+    this.presentationSvg = presentationSvg;
+    this.systemSvg = systemSvg;
+    this.uiSettingsSvg = uiSettingsSvg;
+    this.watchlistSvg = watchlistSvg;
+  }
+  
+  getCustomIcon(iconName) {
+    var customIcon = this.administrationSvg;
+    switch (iconName) {
+      case "svg_administration":
+        customIcon = this.administrationSvg;
+        break;
+      case "svg_automation":
+        customIcon = this.automationSvg;
+        break;
+      case "svg_configuration":
+        customIcon = this.configurationSvg;
+        break;
+      case "svg_data_center":
+        customIcon = this.dataCenterSvg;
+        break;
+      case "svg_data_collection":
+        customIcon = this.dataCollectionSvg;
+        break;
+      case "svg_data_point_details":
+        customIcon = this.dataPointDetailsSvg;
+        break;
+      case "svg_data-distribution":
+        customIcon = this.dataDistributionSvg;
+        break;
+      case "svg_events":
+        customIcon = this.eventsSvg;
+        break;
+      case "svg_help":
+        customIcon = this.helpSvg;
+        break;
+      case "svg_presentation":
+        customIcon = this.presentationSvg;
+        break;
+      case "svg_UI_settings":
+        customIcon = this.uiSettingsSvg;
+        break;
+      case "svg_Watchlist":
+        customIcon = this.watchlistSvg;
+        break;
+      case "svg_system":
+        customIcon = this.systemSvg;
+        break;
+      default:
+        customIcon = this.administrationSvg;
+        break;
+    }
+    return customIcon;
```

Comment:
I've got 2 notes @dailenisgonzalez:
1.  we shouldn't be using `var`
2. you could use an object rather than a switch statement since switch statements aren't easily maintainable and less performant. I suggest something like... 

```
const SVG_ICONS = {
   'SVG_ICON_NAME': this.[SVG_ICON],
   'svg_administration': this.administrationSvg
}

...

getCustomIcon(iconName) {
    return iconName ? SVG_ICONS[iconName] : SVG_ICONS['svg_administration']
}
```